web/src/components/repo/build/BuildLogs.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="build" class="bg-gray-700 dark:bg-dark-gray-700 p-4">
+  <div v-if="build" class="build-logs-container bg-gray-700 dark:bg-dark-gray-700 p-4">
```

Comment:
We use windicss (similar to tailwind) for styling: https://windicss.org/utilities/typography.html#font-family

```suggestion
  <div v-if="build" class="font-mono bg-gray-700 dark:bg-dark-gray-700 p-4">
```

Reply:
Nice, even cleaner!